source/agora/network/NetworkClient.d

```diff
@@ -239,7 +239,7 @@ class NetworkClient
 
     public void sendEnvelope (SCPEnvelope envelope) nothrow
     {
-        this.taskman.runTask(
+        this.taskman.setTimer(0.seconds,
```

Comment:
Could you add a note here?

```suggestion
        // simulates schedule() under the hood
        this.taskman.setTimer(0.seconds,
```

Also as I mentioned, this works for LocalRest. However I am not sure if it works for vibe.d. It would be good to look into whether vibe.d does anything special with zero-timeout timers (it might possibly trigger a context switch right away).

And if we need to fix vibe.d to add a `schedule()` primitive then we should file a tracking issue for it.

Reply:
I will check in vibe.d

Reply:
I could see no special treatment of the case where the timeout is zero in vibe.d core.